### PR TITLE
Adding Crossmint on Polygon's PoS Docs under Deploy

### PR DESCRIPTION
**Pull request by rohit-710**
I work as a DevRel Engineer at Crossmint and I am trying to make a PR to include Crossmint on Polygon's Pos Docs under Deploy. 

Crossmint: http://crossmint.com/

Please review and merge accordingly. Please get back to me if there are any issues with this.

### CODE REVIEW

**Comment by rohit-710** [comment]
@cerberushades can you review this please?

**Comment by rohit-710** [comment]
Hey @cerberushades 
I think it would be nice to get the PR out for now, since I the rest of the deploy docs that I saw had a similar RMD structure. I am happy to help modify it to fit the new structure. 

Essentially we wanna get this merged so people can know that we exist and the tools we provide

**Comment by rohit-710** [comment]
Hey @cerberushades 
Any update here?

File: docs/pos/deploy/crossmint.md [hunk 1]
```diff
@@ -26,13 +26,13 @@ To create a new smart contract using Crossmint's Developer Console, follow the s
 
 1. Navigate to the [Crossmint's Developer Console page:](https://staging.crossmint.com/console/overview/?utm_source=backlinks)
 
-  ![img](../../static/img/Crossmint/screenshot1.jpg)
+  ![img](../../../static/img/Crossmint/screenshot1.jpg)
```
Comment (cerberushades) [inline docs/pos/deploy/crossmint.md:29]:
```suggestion
  ![img](/img/Crossmint/screenshot1.jpg)
```

This path should work. The same for the others.

**Review by cerberushades — CHANGES_REQUESTED** [review]

**Comment by cerberushades** [comment]
> Hey @cerberushades Any update here?

requestted a few more changes. I also dont see the update on sidebars.js.

**Comment by rohit-710** [comment]
Hey @cerberushades 
Thanks for the callout! I just updated those as well, let me know if there are any other changes that I may have missed

**Review by cerberushades — CHANGES_REQUESTED** [review]

**Comment by rohit-710** [comment]
@cerberushades, added!

**Comment by rohit-710** [comment]
I think it should work as expected now, apologies!

**Comment by cerberushades** [comment]
> @cerberushades I think it should work as expected now, apologies!
I'll merge it now. A reminder, though: as I said previously, we're revamping the docs and still discussing if/how we'll be presenting 3rd party tutorials on the new docs. Stay tuned!

**Pull request merged**